---
This is not meant to be pushed, but you're going to need this patch
to run glibc on the current builds of gnumach.

 sysdeps/mach/hurd/x86_64/vm_param.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sysdeps/mach/hurd/x86_64/vm_param.h b/sysdeps/mach/hurd/x86_64/vm_param.h
index f269afb3..e35596bd 100644
--- a/sysdeps/mach/hurd/x86_64/vm_param.h
+++ b/sysdeps/mach/hurd/x86_64/vm_param.h
@@ -19,6 +19,7 @@
 #define _X86_64_VM_PARAM_H
 
 /* Arbitrary start of the brk. This is after usual binary and library mappings.  */
-#define BRK_START	0x200000000000
+// #define BRK_START	0x200000000000
+#define BRK_START	0x20000000
 
 #endif /* x86_64/vm_param.h */
-- 
2.40.1
